src/helpers/buildAudio.js

```diff
@@ -1,12 +1,20 @@
 /* global Audio */
+const AUDIO_CDN = '//audio.qurancdn.com/';
```

Comment:
wouldn't this be better gotten from .env file?

Reply:
I tried, but on development environment it was null. Spent two hours on this :(